sotodlib/hwp/g3thwp.py

```diff
@@ -850,7 +880,7 @@ def _quad_form(self, quad):
         quad[(quad < 0.5)] = 0
         offset = 0
         logger_bit = True
-        for quad_split in np.array_split(quad, len(quad) / 100):
+        for quad_split in np.array_split(quad, 1 + np.floor(len(quad) / 100)):
```

Comment:
Comment about small bug I found

Previously when len(quad) < 100, np.array_split tries to split array into zero splits and raise error.
I added 1 to avoid error.